cmd/config/udn-density-l3-pods/udn.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: l3-network-{{.Iteration}}
+spec:
+  topology: Layer3
+  layer3:
+      role: Primary
+      subnets:
+        - cidr: 10.132.0.0/16
+          hostSubnet: 24
+      mtu: 1300
```

Comment:
does the MTU have to be 1300? Is there a geneve tunnel in place here?

Reply:
cc @mohit-sheth @venkataanil

Reply:
I don't know the reason why @mohit-sheth kept this. But I assumed it as a geneve network.

Reply:
ack. We can modify later if necessary.